### PR TITLE
[primitives,copy] remove bgr24 to bgrx32 optimized

### PR DESCRIPTION
**Pull request by akallabeth**
the function was buggy, remove until resolved.

### CODE REVIEW

**Comment by akallabeth** [comment]
@jackyzy823 did reproduce this now on my fedora machine.
Could you try the referenced pull and check if that now works?
If yes I´ll check what exactly is going wrong here.

**Review by hardening — COMMENTED** [review]
Looks ok

**Comment by akallabeth** [comment]
@hardening just removed the optimized functions, will refine and readd them once I get confirmation this is actually causing the issue.

**Comment by hardening** [comment]
Perhaps we could use primitives of ffmpeg as we kinda need ffmpeg on most platforms to have a decent FreeRDP...

**Comment by akallabeth** [comment]
@hardening not for copy as far as I can see. (and only as option as it is not available on a lot of platforms by default)

**Comment by jackyzy823** [comment]
Yes, this PR seems working. so far so good.

**Comment by akallabeth** [comment]
@jackyzy823 ok, found the bugs ^^
the `avx2` one is tricky, the shuffle function can not cross 16byte boundaries, so we needed a workaround for that.

File: libfreerdp/primitives/sse/prim_copy_sse4_1.c [hunk 1]
```diff
@@ -63,15 +67,20 @@ static INLINE pstatus_t sse_image_copy_bgr24_bgrx32(BYTE* WINPR_RESTRICT pDstDat
...
-			_mm_storeu_si128(dst, d0);
+			for (; x < width; x += 4)
+			{
+				const __m128i* src = (const __m128i*)&srcLine[(x + nXSrc) * srcByte];
+				__m128i* dst = (__m128i*)&dstLine[(x + nXDst) * dstByte];
+				const __m128i s0 = _mm_loadu_si128(src);
+				const __m128i s1 = _mm_shuffle_epi8(s0, smask);
+				const __m128i s2 = _mm_loadu_si128(dst);
+
+				__m128i d0 = _mm_blendv_epi8(s1, s2, mask);
```
Comment (jackyzy823) [inline libfreerdp/primitives/sse/prim_copy_sse4_1.c:81]: The thing i missed :( ,  Restoring the original alpha value  :+1:

**Comment by jackyzy823** [comment]
I tested and it is so far so good under both sse and avx2. 

Thanks for your work! :+1:

**Pull request merged**